### DISCUSSION TITLE
I am facing issue while adding the `<OrganizationProfile />` component as a Dialog component.

### DISCUSSION THREAD

**Discussion by Nil2000** [clerk/javascript#3710] [Support request - DO NOT OPEN HERE] [answered]
The Member and General tab is disabled for some reason. Can anyone help me regarding this?
![image](https://github.com/user-attachments/assets/171afb68-b078-4efd-a166-ba136a618264)

Package : **clerk/nextjs**
Doc link : [https://clerk.com/docs/components/organization/organization-profile](https://clerk.com/docs/components/organization/organization-profile)

**Comment by jescalan** [comment, marked as answer]
https://github.com/orgs/clerk/discussions/3829

**Comment by songhanpoo** [comment]
same here +1,
I'm implementing the `<OrganizationProfile />` in dialog by shadcn.

![Screenshot 2025-01-24 at 22 48 27](https://github.com/user-attachments/assets/919ed664-9060-46c4-b137-e8293922fbce)

**Reply by jescalan** [reply]
https://github.com/orgs/clerk/discussions/3829
